src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -183,12 +186,11 @@ public DistributedPubSubMediator(DistributedPubSubSettings settings)
             });
             Receive<Publish>(publish =>
             {
-                var topic = Uri.EscapeDataString(publish.Topic);
-                var path = Self.Path / topic;
+                string path = Internal.Utils.MakeKey(Self.Path / Internal.Utils.EncodeName(publish.Topic));
```

Comment:
I would probably have used a `using static Internal.Utils` and just have `MakeKey` and `EnodeName` in the code, as it is easier on the eye and the usage varies between code inside and outside of the `Internal` namespace.